Doc/cmus.txt

```diff
@@ -632,6 +632,9 @@ update-cache [-f]
 	-f
 		Update all files. Same as quit, rm -f $XDG_CONFIG_HOME/cmus/cache, start cmus.
 
+version
+	print the version information
```

Comment:
Sorry to nitpick, but all descriptions should look like full sentences: start with a capital letter and end with a period.